content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,425 @@
+the max page size.
+
+### Posting Documents Using DocumentReference Resource
+
+If your app uses the [DocumentReference resource](http://fhir.cerner.com/millennium/dstu2/infrastructure/document-reference/)
+to post clinical notes back to the system, the following provides some
+additional parameters to follow to ensure that your document is
+displayed as intended.
+
+- **Document Type:** We accept HTML5 conventions and only XHTML
+  formatted documents are supported. Our system converts them to RTF
+  because that is the current requirement for posting to the Cerner EMR.
+  Since documents are being converted from one format type to another,
+  there is potential for loss of fidelity.
+  - A common error we have seen is with the use of `<br>` tags.  In
+    XHTML, they need to be self-closed as follows: `<br />`. This
+    greatly enhances the accuracy our internal conversion tools.
+  - Some sanitization is run on provided XHMTL. For example: `Applet`,
+    `iframe`, `link`, `script`, and `style` tags will be removed
+    completely. Other tags (`a`, `button`, `form`, `frame`, `frameset`,
+    `input`, `object`, `option`, `select`, `textarea`) may be removed
+    but the text within will remain.
+  - Table styling is not fully supported
+
+- **Modifier Elements:** [implicitRules](http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.implicitRules),
+  [modifierExtension](https://www.hl7.org/FHIR/DSTU2/domainresource.html)
+  and [relatesTo](https://www.hl7.org/FHIR/DSTU2/domainresource.html) are
+  not supported and will be rejected if present.
+
+- **Special Formatting:** Please do not use specialized formatting
+  dependent on CSS or JavaScript. This will be stripped out of your
+  document. It is also not recommended that you support multiple
+  languages within your documents, as this has proven to be error prone
+  with our current conversion tools.
+  - If special formatting is needed, we recommend that you post your
+    formatted text as image files within the HTML.
+
+- **Images:** If posting images within your documents or as the documents themselves, external image links will not function as intended. All images must be embedded Base64 encoded files using web-safe image formats due to file size. Examples are as follows:
+
+  ```
+  <img alt="image" src='data:image/png;base64, <-ENCODED IMAGE-> ' /> 
+  ```
+
+- **Testing:** There are several XHTML validators that can be used to
+  test your document. We suggest using XHTML 1.0 strict as the document
+  type to run your validation with. Try using:
+  - [validator.w3.org/#validate_by_upload+with_options](http://validator.w3.org/#validate_by_upload+with_options)
+  - [html5.validator.nu](https://html5.validator.nu/)
+
+### Requests by Specific LOINC Codes
+
+Cerner creates concept mappings as part of our SMART on FHIR
+implementation that follow the [concept map](https://www.hl7.org/fhir/dstu2/conceptmap.html)
```

Comment:
We don't directly expose this resource/standard. I believe this may be trying to say we do this as a process rather than formally following the specification? Perhaps, remove first sentence completely, change other from "The specification maps... " to "Cerner maps proprietary...". Also, loinc should just be an example (ie: "...with standardized codes such as LOINC." - we do this for all concepts/terminologies we expose.